src/GovUk.Education.ExploreEducationStatistics.Data.Processor/FailedImportsHandler.cs

```diff
@@ -33,35 +37,23 @@ public static void CheckIncompleteImports()
 
                 foreach (var entity in resultSegment.Results)
                 {
-                    var lastBatch = ImportStatusService.GetNumBatchesComplete(entity);
-                    if (entity.NumBatches != 1 && entity.NumBatches == lastBatch)
+                    Console.Out.WriteLine($"Recovering {entity.PartitionKey} : {entity.RowKey}");
```

Comment:
Use logger?

Reply:
Can't inject here - don't think logger is available at this point in the startup.